Arrowgene.Ddon.Shared/Model/MailItemState.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Arrowgene.Ddon.Shared.Model
+{
+    public enum MailItemState : byte
```

Comment:
these bit flags can be combined? if so then maybe an enum isn't the best option

Reply:
Unsure. These were the only things I could find which matched the name of the field. In the packets captured by Rumi, I see a value of 3 when there are items to claim and 1 when the items are all claimed.

Reply:
> these bit flags can be combined? if so then maybe an enum isn't the best option

It seems like enums are fine for using as bitwise flags btw. Just, I just use the `[Flags]` attribute with the enum.

- https://stackoverflow.com/a/8480